cmd/manager/main.go

```diff
@@ -254,14 +254,26 @@ func main() {
 		Preflights:         preflights,
 	}
 
+	cm := contentmanager.NewManager(clientRestConfigMapper, mgr.GetConfig(), mgr.GetRESTMapper())
+	contentManagerFinalizerKey := fmt.Sprintf("%s/contentmanager-cleanup", domain)
+	err = clusterExtensionFinalizers.Register(contentManagerFinalizerKey, finalizerFunc(func(ctx context.Context, obj client.Object) (crfinalizer.Result, error) {
+		ext := obj.(*ocv1alpha1.ClusterExtension)
+		cm.Delete(ext)
```

Comment:
Add a context to the interface used here

Reply:
This is an internal API, so actually not. If this were an external API we would have needed to add the context